File: Backend/TravellifeChaser/Models/RACSystem/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TravellifeChaser.Models.RACSystem
{
    public class Car
    {
        public int Id { get; set; }
        public string Model { get; set; }
        public string Mark { get; set; }
        public string Type { get; set; }
        public int Year { get; set; }
        public int Seats { get; set; }
        public double Rating { get; set; }
        public double DailyPrice { get; set; }
        public string Image { get; set; }//u bazi cuvati putanje do slika koje ce biti smestene u folderu Images
        public bool Rented { get; set; }
        public bool IsReservedForRent { get; set; }
        public int RACServiceId { get; set; }
        public bool QuickRented { get; set; }
        public DateTime QuickRentDate { get; set; }
        public int QuickRentDiscount { get; set; }
        public RACService RACService { get; set; }
        public ICollection<Rent> Rents { get; set; } = new List<Rent>(); // N : 1  Rent - Car
    }
}

File: AvioAndRentACarSystem/src/app/ModelRAC/car.ts
import { RACService } from "./racService";

export class Car
{
    id: number;
    model: string;
    mark: string;
    type: string;
    year: number;
    seats: number;
    rating: number;
    dailyPrice: number;
    image: string;
    rented: boolean;
    racService: RACService;
    racServiceId: number;
    isReservedForRent: boolean;
    quickRented: boolean;
    quickRentDate: Date;
    quickRentDiscount: number;

    constructor(id?: number, model?: string, mark?: string, type?: string, year?: number, seats?: number, rating?: number, dailyPrice?: number, image?: string, rented?: boolean, racServiceId?: number )
    {
        this.id = id;
        this.model = model;
        this.mark = mark;
        this.type = type;
        this.year = year;
        this.seats = seats;
        this.rating = rating;
        this.dailyPrice = dailyPrice;
        this.image = image;
        this.rented = rented;
        this.racServiceId = racServiceId;
    }
}